x-pack/plugins/uptime/public/components/monitor/synthetics/step_detail/waterfall/types.ts

```diff
@@ -48,6 +49,9 @@ export const FriendlyTimingLabels = {
       defaultMessage: 'Content downloading',
     }
   ),
+  [Timings.Total]: i18n.translate('xpack.uptime.synthetics.waterfallChart.labels.timings.total', {
```

Comment:
Does this need a new label, or should I use Timings.Receive? Will they ever need different content? Is it best to keep it separate.

Reply:
yes let's keep it seprate